src/validation.cpp

```diff
@@ -1801,7 +1801,7 @@ bool static FlushStateToDisk(CValidationState& state, FlushStateMode mode)
                 return AbortNode(state, "Failed to write to coin database");
             nLastFlush = nNow;
             // Update money supply on memory, reading data from disk
-            if (!ShutdownRequested()) {
+            if (!ShutdownRequested() && !IsInitialBlockDownload()) {
```

Comment:
Could add reindex and importing as well.

Reply:
Both are covered by `IsInitialBlockDownload()` already.

Reply:
yeah, missed it, cool.